packages/bpk-component-floating-notification/src/BpkFloatingNotification.module.scss

```diff
@@ -0,0 +1,97 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index';
+
+.bpk-floating-notification {
+  position: absolute;
+  right: 0;
+  bottom: bpk-spacing-xl();
+  left: 0;
+  display: flex;
+  max-width: $bpk-one-pixel-rem * 400;
```

Comment:
This seems like an extreme value. What is the purpose of this and can we use any breakpoint tokens?
Or at least justify where the value comes from

Reply:
The value comes from the design specification ([see in Figma](https://www.figma.com/file/f83w1PW3MI05c27NlNL84W/Floating-notification-bars?node-id=1388%3A9493)).

I'm not sure how I would specify 400px otherwise. Any ideas?

![image](https://user-images.githubusercontent.com/64750210/174056702-8dd468a7-aab2-497f-8ae4-f92ca6a6adc5.png)

Reply:
Just to make it less random and cleaner lets go for `bpk-spacing-xxl() * 10`?